### ISSUE TITLE
Update dependencies and used node version

### ISSUE THREAD

**Issue by brot** [TheConnMan/docker-hub-rss#41]
The nodejs version used in the Dockerfiles is nodejs 12 which is already EOL (https://endoflife.software/programming-languages/server-side-scripting/nodejs)

The used javascript dependencies could also be updated. e.g docker-hub-api is already deprecated and log4js is outdated

**Comment by TheConnMan** [comment]
I've upgraded to Node 16. Any help on finding updated dependencies would be greatly appreciated. Thanks!

**Comment by brot** [comment]
I looked into the used dependencies, but some of them are deprecated and outdated
* [docker-hub-api](https://www.npmjs.com/package/docker-hub-api) is deprecated and the used version is 4 years old. Did not really find an up-to-date docker-hub API library ☹️
* Not really sure about [log4js](https://www.npmjs.com/package/log4js), but maybe we can use the latest version 6.7.0

I created the PR #43 to upgrade log4js

**Comment by TheConnMan** [comment]
Yeah, I'm not sure we'll get a replacement for `docker-hub-api`...
